searchapp/views.py

```diff
@@ -8,6 +8,7 @@
 def search(request):
     # return render(request, 'homeapp/index.html')
     # return HttpResponse('<h3>search detail for :'+str(city)+str(category)+' </h3>')
+    print "##########################", request
```

Comment:
@singhsterabhi This is for debugging. I think you will know.
whenever you want to test your view. write a print statement like this and when this view is called. you can see the result in terminal like this
![screenshot from 2016-06-25 12 44 53](https://cloud.githubusercontent.com/assets/7015220/16355390/db3af220-3ad2-11e6-9089-ab88b446653c.png)